packages/client/src/pages/CheckerPage.tsx

```diff
@@ -14,16 +14,26 @@ import {
 import { getDataNeed } from "../config/autofill";
 import { actions, eventNames, sections } from "../config/matomo";
 import { DebugDecisionTable } from "../debug";
-import { useChecker, useTopic, useTopicData, useTracking } from "../hooks";
+import {
+  useChecker,
+  useSlug,
+  useTopic,
+  useTopicData,
+  useTracking,
+} from "../hooks";
+import { SessionContext } from "../SessionContext";
 import { isEmptyObject } from "../utils";
+import ErrorPage from "./ErrorPage";
 import LoadingPage from "./LoadingPage";
 
 const CheckerPage = () => {
   const { topicData, setTopicData } = useTopicData();
   const { checker } = useChecker();
   const topic = useTopic();
+  const slug = useSlug();
   const { matomoTrackEvent } = useTracking();
   const { text } = topic;
```

Comment:
You can destructure slug and text directly from useTopic?
`const {slug, text} = useTopic();`
No need for useSlug

Reply:
1. I don't know why, but I cannot get slug from useTopic: the bug with `topicData` returns. I **need** to get slug here from `useSlug`. ❌ 
2. I can destruct `text` from `useTopic` ✅